dev-tools/packer/platforms/dashboards/run.sh.j2

```diff
@@ -10,14 +10,14 @@ if [ "$SNAPSHOT" = "yes" ]; then
     VERSION="${VERSION}-SNAPSHOT"
 fi
 
-mkdir /beats-dashboards-${VERSION}
-cp -a dashboards/. /beats-dashboards-${VERSION}/
-echo "$BUILDID" > /beats-dashboards-${VERSION}/.build_hash.txt
+mkdir /${BEATNAME-beats}-dashboards-${VERSION}
```

Comment:
I think the bracket should close before -beats? We should leave it -beats it think?